components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -3157,7 +3162,11 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
     async setUsageAttribution(ctx: TraceContext, usageAttributionId: string): Promise<void> {
         const user = this.checkAndBlockUser("setUsageAttribution");
         try {
-            await this.userService.setUsageAttribution(user, usageAttributionId);
+            const attrId = AttributionId.parse(usageAttributionId);
+            if (attrId) {
+                await this.userService.setUsageAttribution(user, usageAttributionId);
+                this.messageBus.notifyOnSubscriptionUpdate(ctx, attrId).catch();
```

Comment:
I'm not sure I fully understand the implications of this change, would you be able to give me a TLDR?